solutions/client-landing-zone/setters.yaml

```diff
@@ -135,6 +135,7 @@ data:
     - "debian.org"
     - "deb.debian.org"
     - "ubuntu.com"
+    - "cloud.google.com"
```

Comment:
might be worth adding a comment above this setter for this particular entry?

Reply:
+1

Reply:
included in latest commit